gensim/test/test_logentropy_model.py

```diff
@@ -19,9 +19,39 @@
 
 
 class TestLogEntropyModel(unittest.TestCase):
+    TEST_CORPUS = [[(1, 1.0)], [], [(0, 0.5), (2, 1.0)], []]
+
     def setUp(self):
         self.corpus_small = MmCorpus(datapath('test_corpus_small.mm'))
         self.corpus_ok = MmCorpus(datapath('test_corpus_ok.mm'))
+        self.corpus_empty = []
+
+    def testGeneratorFail(self):
+        # attempt to create model generator
+        def get_generator(test_corpus=TestLogEntropyModel.TEST_CORPUS):
+            for test_doc in test_corpus:
+                yield test_doc
+        try:
+            model = logentropy_model.LogEntropyModel(get_generator())
+        except ValueError:
+            model = 'None'
+        except Exception:
+            model = 'Other'
+        # logentropy_model.LogEntropyModel should throw a value error
```

Comment:
This is a poor comment, because it's just restating what's already obvious from the code. Get rid of t.

Reply:
+1